docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -472,6 +473,32 @@ You can also specify authentication credentials with the proxy host.
 testcafe chrome my-tests/**/*.js --proxy username:password@proxy.mycorp.com
 ```
 
+### --proxy-bypass \<rules\>
+
+Specifies the resources accessed bypassing the proxy server.
+
+When you access the Internet through a proxy server specified using the [--proxy](#--proxy-host) option, you may still need some local or external resources to be accessed directly. In this instance, provide their URLs to the `--proxy-bypass` option.
+
+The `rules` parameter takes a comma-separated list (without spaces) of URLs that require direct access. You can omit parts of the URLs or replace them with wildcards `*` (which has the same effect). TestCafe will correspond these symbols to any number of characters in the URL.
+
+The following example uses the proxy server at `proxy.corp.mycompany.com` with the `localhost:8080` address accessed directly.
+
+```sh
+testcafe chrome my-tests/**/*.js --proxy proxy.corp.mycompany.com --proxy-bypass localhost:8080
+```
+
+In the example below, two resources are accessed bypassing the proxy: `localhost:8080` and `internal-resource.corp.mycompany.com`.
+
+```sh
+testcafe chrome my-tests/**/*.js --proxy proxy.corp.mycompany.com --proxy-bypass localhost:8080,internal-resource.corp.mycompany.com
+```
+
+The `*.mycompany.com` value means that all URLs in `mycompany.com` subdomains will be accessed directly.
```

Comment:
Should we mention that the rule `*.mycompany.com` is the same as `.mycompany.com` (without *) ? I mean that it is not required to add an asterisk in the beginning of in the end of rule, but it's required in the middle of rule (.mycompany.*.com)

Reply:
Yeah, I didn't thing about wildcards in the middle